livekit-plugins/livekit-plugins-speechmatics/livekit/plugins/speechmatics/types.py

```diff
@@ -76,7 +76,7 @@ def asdict(self):
 class ConnectionSettings:
     """Defines connection parameters."""
 
-    url: str
+    url: str = "wss://eu2.rt.speechmatics.com/v2"
```

Comment:
should this default to the EU server? do you happened to have a global endpoint where it would connect to the closest instance from where the agent is running?

Reply:
we don't have a global endpoint. here's the the link to our docs listing all the endpoints https://docs.speechmatics.com/introduction/authentication#supported-endpoints